test/source/tests/setup.ts

```diff
@@ -725,20 +725,20 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
     // todo - change to an "add key" instead of initial import test
     // todo - disable initial import of s/mime key
     // todo - disable import of encrypted s/mime key, require decrypted?
-    // ava.default.only(
-    //   'setup - s/mime private key',
-    //   testWithBrowser(undefined, async (t, browser) => {
-    //     const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'flowcrypt.test.key.imported@gmail.com');
-    //     const key = {
-    //       title: 's/mime pkcs12 encrypted key',
-    //       filePath: 'test/samples/smime/human-unprotected-PKCS12.p12',
-    //       armored: null,
-    //       passphrase: 'test pp to encrypt unprotected key',
-    //       longid: null
-    //     };
-    //     await SetupPageRecipe.manualEnter(settingsPage, key.title, { submitPubkey: false, usedPgpBefore: false, key });
-    //   })
-    // );
+    ava.default(
```

Comment:
@tomholub The test now passes, but we should really throw an exception when trying to import S/MIME key as the only private key? And disable this test again for now?
I'll also create an add key test like this.

Reply:
I guess we should, treating S/MIME private keys as second class citizens.

However, I'd also be ok with a warning (`Ui.modal.warn`) that the user can ignore. `Using S/MIME as the only key on account is experimental and not recommended. Continue anyway? [ok] [cancel]`

Reply:
Looks like Ui.modal.warning isn't designed to have [ok, cancel] options. I implemented that with UI.modal.confirm, ok?

Reply:
sorry for confusion - yes - that's what I meant.